exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -134,15 +179,20 @@ func (prwe *prwExporter) PushMetrics(ctx context.Context, md pmetric.Metrics) er
 
 		tsMap, err := prometheusremotewrite.FromMetrics(md, prwe.exporterSettings)
 		if err != nil {
-			err = consumererror.NewPermanent(err)
+			prwe.telemetry.recordTranslationFailure(ctx)
+			prwe.settings.Logger.Debug("failed to translate metrics %s", zap.Error(err))
+			prwe.settings.Logger.Debug("exporting remaining %s metrics", zap.Int("translated", len(tsMap)))
```

Comment:
```suggestion
			prwe.settings.Logger.Debug("failed to translate metrics, zap.Error(err))
			prwe.settings.Logger.Debug("exporting remaining metrics", zap.Int("count", len(tsMap)))
```

The `%s` is for formatted strings. Zap uses structured logging instead. 

I also wondering if we want to adjust these log lines to be more clear. I think they may confuse users. I think the message intent is

1. Some metrics failed to translate. This was not a catastrophic failure. 
2. N metrics were successfully translated. The PRWE will attempt to export these.